src/program.js

```diff
@@ -473,6 +473,12 @@ Example: $0 --help run.
         demandOption: false,
         type: 'boolean',
       },
+      'reload-delay': {
+        describe: 'Allow web-ext to wait until a WebExtension build has ' +
+                  'completed the building process',
```

Comment:
While this option can be used to support that scenario, this is not an accuate description of this option.